cost/superseded_instance_remediation/superseded_instance_remediation.pt

```diff
@@ -0,0 +1,304 @@
+name "Superseded Instance Remediation"
+rs_pt_ver 20180301
+type "policy"
+short_description "This Policy Template is used to automatically supersede instances based on user-defined standards. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/superseded_instance_remediation/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+category "Cost"
+severity "low"
+
+###############################################################################
+# Permissions
+###############################################################################
+
+permission "perm_instances" do
+  label "Permissions for instances"
+  actions "rs_cm.index", "rs_cm.show"
+  resources "rs_cm.instances"
+end
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses"
+  description "Email addresses of the recipients you wish to notify"
+end
+
+parameter "param_old_instance_type" do
+  label "Disallowed Instance Type"
+  type "string"
+  description "Old Instance Type"
```

Comment:
Is it disallowed or old instance types.  I think they can be different.  Also include a better description of what the parameter is about.